src/components/LandingPage/index.tsx

```diff
@@ -99,7 +99,7 @@ export const LandingPage = () => {
     <PageWrapper>
       <Header>
         <InnerHeader>
-          <LinkCreateNFT to="/mint/steps">{t('mintNFT')}</LinkCreateNFT>
+          <LinkCreateNFT to="/mint/steps/0">{t('mintNFT')}</LinkCreateNFT>
```

Comment:
same point in nft mint. param must specific for id .why we insert 0 as a start point